app/controllers/sign_up/registrations_controller.rb

```diff
@@ -62,5 +66,15 @@ def sp_request_id
 
       ServiceProviderRequestProxy.from_uuid(request_id).uuid
     end
+
+    def redirect_if_ial2_and_vendor_outage
+      return unless ial2_requested?
+
+      vendor_status = VendorStatus.new
+      if vendor_status.any_ial2_vendor_outage?
```

Comment:
same, could inline this if we want

```suggestion
      if VendorStatus.new.any_ial2_vendor_outage?
```